examples/blender/blender.py

```diff
@@ -16,13 +16,16 @@
 import utils  # noqa
 
 
-async def main(subnet_tag="testnet", stream_output=False):
+async def main(subnet_tag="testnet", capture_ctx=None):
     package = await vm.repo(
         image_hash="3c436e6bdfa188e35b2881be6377e41a63062e8cd9710345757d3559",
         min_mem_gib=1.0,
         min_storage_gib=2.0,
     )
 
+    if not capture_ctx:
```

Comment:
```suggestion
    if capture_ctx is None:
```
(Because e.g. `not ""` is True.)

Reply:
IMHO this is perfectly fine

Reply:
Yes, sorry, I didn't notice the `capture_ctx` type.